auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -82,9 +81,25 @@ public Boolean nullable() {
 
   @Override
   public boolean applicable(Context context) {
+    TypeMirror autoValueClass = context.autoValueClass().asType();
+    // Disallow manual implementation of the CREATOR instance
+    if (!needsCreator(context)) {
+      context.processingEnvironment().getMessager().printMessage(Diagnostic.Kind.ERROR,
+          "Manual implementation of a static Parcelable.Creator<T> CREATOR field found when processing "
+              + autoValueClass.toString() + ". Remove this so auto-value-parcel can automatically generate the "
+              + "implementation for you.");
+      return false;
+    }
+    // Disallow manual implementation of writeToParcel
+    if (!needsWriteToParcel(context)) {
+      context.processingEnvironment().getMessager().printMessage(Diagnostic.Kind.ERROR,
```

Comment:
Pass the executable element corresponding to the method as the third arg to this method so the error is associated with it.